.travis_install.bash

```diff
@@ -19,6 +19,16 @@ download-llvm(){
   popd
 }
 
+download-llvm-macos(){
+  echo "Downloading and installing the LLVM specified by envvars..."
+
+  wget "http://releases.llvm.org/${LLVM_VERSION}/clang+llvm-${LLVM_VERSION}-x86_64-apple-darwin.tar.xz"
+  tar -xvf clang+llvm*
+  pushd clang+llvm* && sudo mkdir /tmp/llvm && sudo cp -r ./* /tmp/llvm/
+  sudo ln -s "/tmp/llvm/bin/llvm-config" "/usr/local/bin/${LLVM_CONFIG}"
+  popd
```

Comment:
Iff this doesn't work quite right (maybe rpath bugs?), then something like `tar xJf - --strip-components 1 -C /usr/local/ clang+llvm-${LLVM_VERSION}-x86_64-apple-darwin` will just stick it all in `/usr/local` and has appeared to work locally for me.  😜